monitor/dbt_project/macros/get_adapter_type_and_unique_id.sql

```diff
@@ -1,5 +1,5 @@
-{% macro get_adapter_unique_id() %}
-    {{ elementary.edr_log(adapter.dispatch('get_adapter_unique_id')()) }}
+{% macro get_adapter_type_and_unique_id() %}
+    {{ elementary.edr_log(tojson([target.type, adapter.dispatch('get_adapter_unique_id')()])) }}
```

Comment:
NIT:
Do we prefer list over dict?

Reply:
Following the convention of `get_elementary_database_and_schema`.  
Also, this makes it possible to do `adapter_type, adapter_unique_id = self.dbt_runner.run_operation('get_adapter_type_and_unique_id')` which is kinda nice.